src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -382,6 +383,24 @@ public void holdCurrentPosition(){
     setBaseReference(getBaseAngle());
     setElbowReference(getElbowAngle());
   }
+
+  public void checkAngles() {
+    if(getBaseAngle() > kMaxBaseAngle || 
+    getBaseAngle() < kMinBaseAngle || 
+    getChooChooAngle() > kMaxChooChooAngle ||
+    getChooChooAngle() < kMinChooChooAngle ||
+    getElbowAngle() > kMaxElbowAngle ||
+    getElbowAngle() < kMinElbowAngle) {
+      if(DriverStation.isDisabled()) {
+        DriverStation.reportError("Encoders read outside of possible positions, check your offsets!", true);
+      } else {
+        String evilString = null;
+        evilString.toString();
```

Comment:
Since this is an odd case, its worth adding code comments that it is intentional

Reply:
done, comments have now been added